components/SignUpResources/SignUpResources.tsx

```diff
@@ -12,7 +12,7 @@ const resourceTypeBuilder = (id: string) => {
   const dictionary = {
     services: () => <SignUpServicesFormWithModal />,
     products: () => <SignUpProducts />,
-    volunteer: () => <SignupVolunteering />,
+    "signup.volunteering.header": () => <SignupVolunteering />,
```

Comment:
do not change the dictionary because of possible side effects
Let's keep that key (`volunteer`) and change the value of it in the RO and UA locale from "Voluntar" to "Voluntariat"  
I saw that for other languages it's ok (EN and RU)